dali/python/nvidia/dali/ops.py

```diff
@@ -659,9 +672,9 @@ def _check_common_length(self, inputs):
             for input in inputs:
                 if isinstance(input, list):
                     if len(input) != arg_list_len:
-                        raise ValueError(("All argument lists for Multiple Input Sets used " +
-                                          "with operator {} must have the same length")
-                                          .format(type(self).__name__))
+                        raise ValueError("All argument lists for Multiple Input Sets used " +
+                                         "with operator {} must have the same length"
+                                         .format(type(self).__name__))
```

Comment:
Binary op at the end of line? It shouldn't have passed through flake8...
```suggestion
                        raise ValueError(f"All argument lists for Multiple Input Sets used "
                                         f"with operator {type(self).__name__} must have "
                                         f"the same length")
```

Reply:
Done.

By the way, it can pass the flake8. From PEP 8:

> In Python code, it is permissible to break before or after a binary operator, as long as the convention is consistent locally. 

They encourage breaking before the binary operator but there is no strong check.